### ISSUE TITLE
labels_for_lpa initialization

### ISSUE THREAD

**Issue by Algebra-FUN** [Vimalakirti/GCN-LPA-PyTorch#4]
The first problem:
```python
if args.cuda:
    model.to(device)
    features = features.to(device)
    idx_test = idx_test.to(device)
    labels_for_lpa = one_hot_embedding(labels, labels.max().item() + 1).type(torch.FloatTensor).to(device)
```
You only initialize `laberls_for_lpa` when cuda is used.

Another problem:
According to the paper, the label for unlabeled nodes (not for train) should be set as zero vectors. I don't see this processing in your code
